### ISSUE TITLE
[Spree Upgrade] Fix admin/variant_overrides_spec (1 broken test)

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#3112]
Fix admin/variant_overrides_spec

```
3) 
  As an Administrator
  With products I can add to my hub's inventory
  I want to override the stock level and price of those products
  Without affecting other hubs that share the same products
 as an enterprise user selecting a hub displays a list of hub choices (ie. only those managed by the user)
     Failure/Error: click_link 'Inventory'
     
     Capybara::ElementNotFound:
       Unable to find visible link "Inventory"
     # ./spec/features/admin/variant_overrides_spec.rb:37:in `block (4 levels) in <top (required)>'

  4) 
  As an Administrator
  With products I can add to my hub's inventory
  I want to override the stock level and price of those products
  Without affecting other hubs that share the same products
 as an enterprise user when inventory_items exist for variants when a hub is selected with no overrides displays the list of products with variants
     Failure/Error: within "tr#v_#{variant.id}" do click_button 'Hide' end
     
     Capybara::ElementNotFound:
       Unable to find visible button "Hide" within #<Capybara::Node::Element tag="tr" path="//HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[3]/DIV[1]/DIV[1]/DIV[1]/DIV[1]/DIV[5]/FORM[1]/TABLE[1]/TBODY[1]/TR[2]">
     # ./spec/features/admin/variant_overrides_spec.rb:120:in `block (7 levels) in <top (required)>'
     # ./spec/features/admin/variant_overrides_spec.rb:120:in `block (6 levels) in <top (required)>'

    6) 
  As an Administrator
  With products I can add to my hub's inventory
  I want to override the stock level and price of those products
  Without affecting other hubs that share the same products
 as an enterprise user when inventory_items do not exist for variants when a hub is selected alerts the user to the presence of new products, and allows them to be added or hidden
     Failure/Error: within "table#new-products tr#v_#{variant2.id}" do click_button 'Hide' end
     
     Capybara::ElementNotFound:
       Unable to find visible button "Hide" within #<Capybara::Node::Element tag="tr" path="//HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[3]/DIV[1]/DIV[1]/DIV[1]/DIV[1]/DIV[5]/TABLE[1]/TBODY[1]/TR[2]">
     # ./spec/features/admin/variant_overrides_spec.rb:368:in `block (6 levels) in <top (required)>'
     # ./spec/features/admin/variant_overrides_spec.rb:368:in `block (5 levels) in <top (required)>'
```

**Comment by luisramos0** [comment]
First test will be fixed in #3119

**Comment by HugsDaniel** [comment]
Hey @luisramos0 for the first failing spec, it's not related to internationalisation. The tab "Inventory" is named "Variant Overrides" in 2.0

**Comment by luisramos0** [comment]
#3171 fixed the 2nd and 3rd broken specs in this issue. One remaining, to be fixed in #3119
